src/index.html

```diff
@@ -1,14 +1,51 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <title>Moyo Header</title>
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="icon">
+        <img src="images/logo.png" alt="Moyo">
+      </a>
+      <nav class="navigation">
+        <ul class="ul-navigation">
+          <li class="nav_items">
+            <a href="#" class="is-active nav__links">Apple</a>
+          </li>
+          <li class="nav_items">
```

Comment:
Add blank line between the links
```suggestion
          </li>
          
          <li class="nav_items">
```